processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorUnit.java

```diff
@@ -66,11 +66,13 @@ private void processTask(TaskRequest request) {
         Timer timer = Utils.timer();
         try {
             pipeline.scheduleThenProcess(request);
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            log.warn("Scheduling and/or processing interrupted. Shutdown initiated={}, request: {}",
+                     terminated, request);
```

Comment:
Isn't better to pass `e` ?

Reply:
I'm not sure why we'd want to log what sleep was interrupted - does it help?